src/python/library/tritonclient/http/aio/__init__.py

```diff
@@ -51,8 +51,12 @@ async def _get_error(response):
     """
     if response.status != 200:
         result = await response.read()
-        error_response = json.loads(result) if len(result) else {"error": ""}
-        return InferenceServerException(msg=error_response["error"])
+        try:
+            error_response = json.loads(result)
```

Comment:
Make this behaviour consistent with http client.

Reply:
> > Thanks for submitting this pull request! Can you please submit a Contributor License Agreement? https://github.com/triton-inference-server/server/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla
> > Once done, we can review the PR.
> 
> I have already submitted that for a previous contribution, do I need a new one?

Apologies, GitHub marked this as your first PR. Forgot about your previous one. Thanks, we'll review shortly.